tests/x11regressions/libreoffice/libreoffice_mainmenu_favorites.pm

```diff
@@ -22,8 +23,10 @@ sub run {
     assert_screen('test-desktop_mainmenu-1');
 
     # find the favorites button
-    assert_and_click('application-menu-favorites');
-    assert_screen('menu-favorites-libreoffice');
+    if (!sle_version_at_least('15')) {
```

Comment:
Same as above for `is_sle` :)

Reply:
Thanks, I just fixed it, please help to review.